lib/controller/UserController.js

```diff
@@ -33,12 +33,13 @@ UserController.push({
       query: {
         limit: Joi.number().integer().min(0).required(),
         offset: Joi.number().integer().min(0).required(),
+        search: Joi.string().allow(''),
       },
     },
   },
   handler: async (request) => {
-    const { limit: pageLimit, offset: pageOffset } = request.query;
-    const users = await UserDAO.getUsersPagination(pageLimit, pageOffset);
+    const { limit, offset, search } = request.query;
```

Comment:
This still works, right? 😄 I usually only rename destructured variables when I have to... haha

Reply:
Yeah this should still work. I didn't know at the time that we could simple use the variable names provided when de-structuring 😅